udata/core/spatial/translations.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Here to force translations by gettext
+_ = lambda s: s
+
+TRANSLATIONS = (
```

Comment:
Oh OK I get it, so it's not exactly dynamic. How can we maintain that list? Is there a way to check that we have all entries with a command?